Master/appstats.js

```diff
@@ -18,7 +18,7 @@ const azureStorage = require('azure-storage');
 const m_alUtil = require('../lib/al_util');
 
 const STATS_PERIOD_MINUTES = 15;
-const APP_FUNCTIONS = ['Master', 'O365WebHook', 'Updater'];
+const APP_FUNCTIONS = ['Master', 'O365Collector', 'O365Poller', 'Updater'];
```

Comment:
Sort of meta discussion - If Push method is not an option for Azure, do we still benefit from deploying AzureFunciton into customer's environment?

Reply:
@alexturkin Are you proposing a kind of "Hawk for O365" ?

Reply:
@alexturkin this is the current architecture vision.